src/presence/presence-handler.js

```diff
@@ -120,6 +120,31 @@ module.exports = class PresenceHandler {
 	}
 
 	/**
+	 * Handles unsubscriptions to client login and logout events
+	 *
+	 * @param   {Object} socketWrapper the socketWrapper of the client that left
+	 * @param   {String} actions either a presence leave or join message
+	 *
+	 * @private
+	 * @returns {void}
+	 */
+	_handleUnsubscribe( socketWrapper, action ) {
+		if( action === C.ACTIONS.PRESENCE_JOIN ) {

```

Comment:
i would argue that the front end would only do a subscribe, which means it would receive join and leave events, but it doesn't subscribe to each individually. Seems like an odd case to only be permission to be told when people leave.